src/analytics/HyperGeom.js

```diff
@@ -95,9 +95,11 @@ const HyperGeom = () => {
     return Math.min(Math.max(val, min), max);
   };
 
-  const calculate = (N, S, n, s) => {
-    const keys = [...Array(parseInt(s, 10) + 1).keys()];
-    const values = keys.map((x) => hyp(parseInt(N, 10), parseInt(S, 10), parseInt(n, 10), x));
+  const calculate = (populationSize, sampleSize, popSuccesses, sampleSuccesses) => {
```

Comment:
I changed the names of these arguments to be readable, since that is where the error came from.

Reply:
Whoops that introduced a lint error due to duplicate variable names.
I moved the calculation code to a util file.